Bottom: fc96efd9ca533bb3c515c1f03319ead1d9f8b91a
Top:    427327f6d7f80a4a3fd819feac923c9113a18154
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-07 11:34:58 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index 6a89d41..d606273 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -971,15 +971,15 @@ int fstat(int socket, struct stat *buf)
 {
 	int fd, ret;
 
-	fprintf(fout, "fstat");
+	fprintf(fout, "fstat socket %d\n", socket);
 	if (fd_get(socket, &fd) == fd_rsocket) {
 		ret = real.fstat(socket, buf);
-		fprintf(fout, "fstat - rsocket %d", ret);
+		fprintf(fout, "fstat - rsocket %d\n", ret);
 		if (!ret)
 			buf->st_mode = (buf->st_mode & ~S_IFMT) | __S_IFSOCK;
 	} else {
 		ret = real.fstat(fd, buf);
-		fprintf(fout, "fstat - normal %d", ret);
+		fprintf(fout, "fstat - normal %d\n", ret);
 	}
 	fflush(fout);
 	return ret;
